docs/RULES.md

```diff
@@ -1030,15 +1172,19 @@ Aliases: no-space-in-code
 This rule is triggered on code span elements that have spaces right inside the
 backticks:
 
-    ` some text `
+```markdown
+` some text `
 
-    `some text `
+`some text `
 
-    ` some text`
+` some text`
+```
 
 To fix this, remove the spaces inside the codespan markers:
 
+```markdown
     `some text`
+```
```

Comment:
I believe this should be un-indented.

Reply:
Thank you for recognizing this omission, it has been corrected.